roles/dtc/common/templates/ndfc_route_control/ndfc_route_control.j2

```diff
@@ -0,0 +1,26 @@
+{% for policy_types in group_item %}
```

Comment:
If this is the main, top-level route_control template, please let it sit at the root of the ```template``` directory

Reply:
moved